boards/arm/sam34/arduino-due/README.txt

```diff
@@ -865,5 +872,5 @@ Configuration sub-directories
   nsh-leds:
     This configuration directory will build the NuttX Shell and enable the user
     LEDS (/dev/userleds). It will also enable the LED example program (leds).
-    Running the leds command will start up an LED daemon which will light up the 
-    L (user), TX, and RX LEDs in a binary sequence. 
\ No newline at end of file
+    Running the leds command will start up an LED daemon which will light up the
+    L (user), TX, and RX LEDs in a binary sequence.
```

Comment:
```suggestion
    L (user), TX, and RX LEDs in a binary sequence.

```

Reply:
If I open a file diff tab I see that "newline at the end of file" is missing, so the suggestion is to add that as I think most of the files have it. But I'm not sure if we have any mentions about this in docs.